runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -871,12 +934,118 @@ Java_jdk_internal_misc_Unsafe_registerNatives(JNIEnv *env, jclass clazz)
 	}
 }
 
+/*
+ * Determine if memory addresses overlap.
+ * Will always return false if either object is an array.
+ *
+ * @param sourceObject
+ * @param sourceOffset memory address
+ * @param destObject
+ * @param destOffset memory address
+ * @param actualCopySize size of memory after address
+ * @return true for no memory overlap, otherwise false
+ */
+jboolean
+ memOverlapIsNone(j9object_t sourceObject, UDATA sourceOffset, j9object_t destObject, UDATA destOffset, UDATA actualCopySize) {
+	if (sourceObject == NULL && destObject == NULL) {
```

Comment:
A couple of nitpicks:
* `(sourceObject == NULL && destObject == NULL)` -> `((sourceObject == NULL) && (destObject == NULL))`
Extra brackets means no one has to worry about order of operations.  One less thing for readers of the code to have to remember.

* better to have a single return point.  It always setting a single breakpoint to catch all returns from the function when debugging.

Reply:
I've applied these changes